bzkanban.js

```diff
@@ -1362,6 +1362,7 @@ function showBugModal(bugCurrent, bugUpdate) {
     meta.className = 'bug-meta';
 
     body.appendChild(comments);
+    showSpinner();
```

Comment:
Ah Ok, was just the position of it that threw me off. I think it'd be better to do it before the `httpGet` in that function bc I don't think we need a spinner for the "else" case (i.e. when a card is dragged).